build/Dockerfile

```diff
@@ -137,15 +138,16 @@ apt-get install -y --no-install-recommends \
     xxd
 EOF
 
+# Install Rust to compile foundry from source because we use a specific tag.
+ARG RUST_VERSION
+RUN curl https://sh.rustup.rs -sSf | bash -s -- --default-toolchain ${RUST_VERSION} -y
+ENV PATH="/root/.cargo/bin:${PATH}"
+
 # Install Foundry.
 RUN curl -L https://foundry.paradigm.xyz | bash
-ENV PATH="~/.foundry/bin:${PATH}"
+ENV PATH="/root/.foundry/bin:${PATH}"
 ARG FOUNDRY_COMMIT_VERSION
-RUN <<EOF
-set -e
-bash -c foundryup -C ${FOUNDRY_COMMIT_VERSION}
-ln -s ~/.foundry/bin/anvil /usr/bin/anvil
-EOF
+RUN foundryup -C ${FOUNDRY_COMMIT_VERSION}
```

Comment:
We could do this as an alternative: https://github.com/sunodo/sunodo/blob/ecaddc21084cecfa2df1394969a334ef2777e847/packages/anvil/Dockerfile#L11-L14

Reply:
Be careful with the version sha you pick.
Foundry has a "monthly nightly version" that they keep around.
Others are deleted.